backend/src/main/java/se/chalmers/dat265/group1/api/order/OrderController.java

```diff
@@ -0,0 +1,97 @@
+package se.chalmers.dat265.group1.api.order;
+
+import se.chalmers.dat265.group1.api.ApiController;
+import se.chalmers.dat265.group1.model.Order;
+import se.chalmers.dat265.group1.model.dbEntities.OrderData;
+import se.chalmers.dat265.group1.model.dbEntities.OrderedPart;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class OrderController extends ApiController implements OrderAPI {
+    public OrderController(boolean debug) {
+        super(debug);
+    }
+
+    @Override
+    public List<Order> getAllOrders() {
+        List<OrderData> orderDataList = orderDataRepository.getObjects();
+        return getMerged(orderDataList);
+
+    }
+
+    private List<Order> getMerged(List<OrderData> orderDataList) {
+        List<Order> orders = new LinkedList<>();
+        for (OrderData orderData : orderDataList) {
+            Order temp = new Order(orderData, orderedPartRepository.getObjects("orderID=" + orderData.getId()));
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "orderID=" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)